src/pages/jeonyongmin/Main/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './NavBar.scss';
+
+class NavBar extends React.Component {
+  render() {
+    return (
+      <>
+        <header className="headerYM">
+          <nav className="headerNav">
+            <div className="headerNavLeftSide">
+              <Link to="/loginym">
+                <img
+                  alt="instagram_imoticon"
+                  src="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Instagram_font_awesome.svg/512px-Instagram_font_awesome.svg.png"
+                  className="headerNavLeftSideImg"
+                />
+              </Link>
+              <h1>
+                <a href="#" className="headerNavLeftSideTitle">
+                  Instagram
+                </a>
+              </h1>
+            </div>
+            <div className="headerNavCenter">
+              <input
+                type="text"
+                className="headerNavInput"
+                placeholder="검색"
+              />
+              <i className="fas fa-search"></i>
+            </div>
+            <div className="headerNavRightSide">
+              <a href="#">
+                <img
+                  alt="explore right menu"
+                  src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png"
+                  className="headerNavRightSideImg1"
+                />
+              </a>
+              <button>
+                <img
+                  alt="heart"
+                  src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png"
+                  className="headerNavRightSideImg2"
+                />
+              </button>
+              <button>
+                <img
+                  alt="mypage"
+                  src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/profile.png"
+                  className="headerNavRightSideImg3"
+                />
```

Comment:
map으로 중복 해결가능해 보입니다!

Reply:
넵 첫번째 태그는 링크태그이기에, 단순 버튼태그인 두번째와 세번째를 mapping하여 수정했습니다!
```suggestion
           <a href="#">
              <img
                alt="explore right menu"
                src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png"
                className="headerNavRightSideImg1"
              />
            </a>
            {headerNavRightSideLastTwoButton.map(button => {
              const { id, imgSrc, alt, className } = button;
              return (
                <button>
                  <img alt={alt} src={imgSrc} className={className} key={id} />
                </button>
              );
            })}
```